### PR TITLE
docs(srv): add jobs billing to faq MTA-4127

### PR DESCRIPTION
**Pull request by SamyOubouaziz**
### Your checklist for this pull request

- [x] Name your pull request according to the [contribution guidelines](https://github.com/scaleway/docs-content/blob/main/docs/CONTRIBUTING.md).

### Description

Please describe what you added or changed.

### CODE REVIEW

File: faq/serverless-jobs.mdx [hunk 1]
```diff
@@ -17,6 +17,69 @@ Scaleway Serverless Jobs is a fully managed service that enables efficient execu
...
+| Monthly duration           | 432 000 s (120h) |
+| Amount of memory allocated | 2048 MB          |
+| Amount of vCPU allocated   | 1120 mVCPU       |
+| Free tier                  | Yes              |
+
+#### Price calculation
+
+* **Memory consumption**
+    * *Allocated Memory conversion:* 2048 MB = 2 GB
+    * *Resources consumption:* 432 000 s * 2 GB = 864 000 GB/s
```
Comment (RoRoJ) [inline faq/serverless-jobs.mdx:66]:
```suggestion
    * *Resource consumption:* 432 000 s * 2 GB = 864 000 GB/s
```

File: faq/serverless-jobs.mdx [hunk 2]
```diff
@@ -17,6 +17,69 @@ Scaleway Serverless Jobs is a fully managed service that enables efficient execu
...
+    * *Free tier:* 400 000 GB/s
+    * *Billed resources:* 864 000 - 400 000 = 464 000 GB/s
+    * *Cost:* 464 000 * €0.0000010 = **€0,464**
+* **vCPU consumption**
+    * *Allocated vCPU conversion:* 1120mVCPU = 1.12 vCPU
+    * *Resources consumption:* 432 000 s * 1.12 vCPU =  483 840 vCPU/s
```
Comment (RoRoJ) [inline faq/serverless-jobs.mdx:72]:
```suggestion
    * *Resource consumption:* 432 000 s * 1.12 vCPU =  483 840 vCPU/s
```

File: faq/serverless-jobs.mdx [hunk 3]
```diff
@@ -17,6 +17,69 @@ Scaleway Serverless Jobs is a fully managed service that enables efficient execu
...
+    * *Free tier:* 200 000 vCPU/s
+    * *Billed resources:* 483 840 - 200 000 = 283 840 vCPU/s
+    * *Cost:* 283 840 * €0.0000100 = **€2,8384**
+
+**Monthly Cost: €3,30**
```
Comment (RoRoJ) [inline faq/serverless-jobs.mdx:77]:
```suggestion
**Monthly Cost: €3.30**
```

File: faq/serverless-jobs.mdx [hunk 4]
```diff
@@ -17,6 +17,69 @@ Scaleway Serverless Jobs is a fully managed service that enables efficient execu
...
+    * *Resources consumption:* 432 000 s * 2 GB = 864 000 GB/s
+    * *Free tier:* 400 000 GB/s
+    * *Billed resources:* 864 000 - 400 000 = 464 000 GB/s
+    * *Cost:* 464 000 * €0.0000010 = **€0,464**
+* **vCPU consumption**
+    * *Allocated vCPU conversion:* 1120mVCPU = 1.12 vCPU
+    * *Resources consumption:* 432 000 s * 1.12 vCPU =  483 840 vCPU/s
+    * *Free tier:* 200 000 vCPU/s
+    * *Billed resources:* 483 840 - 200 000 = 283 840 vCPU/s
+    * *Cost:* 283 840 * €0.0000100 = **€2,8384**
```
Comment (RoRoJ) [inline faq/serverless-jobs.mdx:75]:
```suggestion
    * *Cost:* 283 840 * €0.0000100 = **€2.8384**
```

Comment (RoRoJ) [reply faq/serverless-jobs.mdx:75]: I wonder if displaying the price rounded to the closest cent would be clearer? Maybe in brackets?

Comment (RoRoJ) [inline faq/serverless-jobs.mdx:75]:
```suggestion
    * *Cost:* 283 840 * €0.0000100 = **€2.8384**
```

File: faq/serverless-jobs.mdx [hunk 5]
```diff
@@ -17,6 +17,69 @@ Scaleway Serverless Jobs is a fully managed service that enables efficient execu
...
+| Free tier                  | Yes              |
+
+#### Price calculation
+
+* **Memory consumption**
+    * *Allocated Memory conversion:* 2048 MB = 2 GB
+    * *Resources consumption:* 432 000 s * 2 GB = 864 000 GB/s
+    * *Free tier:* 400 000 GB/s
+    * *Billed resources:* 864 000 - 400 000 = 464 000 GB/s
+    * *Cost:* 464 000 * €0.0000010 = **€0,464**
```
Comment (RoRoJ) [inline faq/serverless-jobs.mdx:69]:
```suggestion
    * *Cost:* 464 000 * €0.0000010 = **€0.464**
```

**Review by thomas-tacquet — APPROVED** [review]

**Review by bene2k1 — APPROVED** [review]

**Review by nerda-codes — APPROVED** [review]

**Pull request merged**